src/bin/basic.rs

```diff
@@ -20,7 +20,7 @@ fn build_ui(application: &gtk::Application) {
     window.set_position(gtk::WindowPosition::Center);
     window.set_default_size(350, 70);
 
-    window.connect_delete_event(move |win, _| {
+    window.connect_delete_event(|win, _| {
         win.destroy();
```

Comment:
You probably want app.quit() here?

Reply:
I want remove this call completely: on windows it seems works same.
When main window closed it will be destroyed and application quits.
Ex. no connecting to "delete" event in https://gitlab.gnome.org/GNOME/gtk/tree/master/examples
and `g_application_quit` called only from normal action (timeouts, clicks, network commands)

Reply:
Hm, I checked wrong event: we use "delete-event", but it even don't found in gtk or glib sources while `connect_delete_event` still called.

Reply:
yes, I thought that it don't needed as not in demos\examples
but I also have only gtk4 locally too, so not sure now.